test/e2e/framework/test_context.go

```diff
@@ -184,6 +184,8 @@ type NodeKillerConfig struct {
 	JitterFactor float64
 	// SimulatedDowntime is a duration between node is killed and recreated.
 	SimulatedDowntime time.Duration
+	// StopCh is a channel that is used to notify NodeKiller to stop killing nodes.
+	StopCh chan struct{}
```

Comment:
This should be more specific, `NodeKillerStopCh`.

Reply:
Changed it.